recipe/meta.yaml

```
@@ -36,7 +36,7 @@ requirements:
     - async-lru >=1.0.0
     - httpx >=0.25.0
```

```suggestion
    - httpx >=0.25.0,<1
```
see https://github.com/jupyterlab/jupyterlab/blame/main/pyproject.toml#L39